python/lsst/ts/externalscripts/maintel/make_lsstcam_calibrations.py

```diff
@@ -104,6 +109,27 @@ async def start_remotes(self):
             )
             await self._ocps_group.start_task
 
+        if not hasattr(self, "_mtcs") or self._mtcs is None:
+            self._mtcs = MTCS(domain=self.domain, log=self.log)
```

Comment:
So, the way `MTCS` is defined here it will subscribe to all the topics from all the components. This is usually pretty heavy. To limit resources you can provide a `intended_usage`, something like:

```
self._mtcs = MTCS(
    domain=self.domain,
    log=self.log,
    intended_usage=MTCSUsages.Slew,
)
```

you will have to import `MTCSUsages`,

```
from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
```